tracer/src/Datadog.Trace/SpanTypes.cs

```diff
@@ -57,6 +57,21 @@ public static class SpanTypes
         /// </summary>
         public const string Test = "test";
 
+        /// <summary>
+        /// The span type for a Test Suite integration.
+        /// </summary>
+        public const string TestSuite = "test_suite_end";
```

Comment:
Do these need to be `public`?

Reply:
Nope, I made the change to internal in [b4e5466](https://github.com/DataDog/dd-trace-dotnet/pull/3276/commits/b4e546630340e86717b34b97bcd3b66044413846)